src/utils/page.ts

```diff
@@ -55,9 +55,7 @@ function _collectElementsText(contextSelectors: string, contextElementsLength: n
 
 function encodedByteCount(s: string): number {
   // From: https://stackoverflow.com/questions/2219526/how-many-bytes-in-a-javascript-string
-  const utf8Bytelength = encodeURI(s).split(/%..|./).length - 1
-  const base64EncodedLength = 4 * Math.ceil(utf8Bytelength / 3.0)
-  return base64EncodedLength
+  return new Blob([s]).size
```

Comment:
It changes the semantics of this function: it first counted how many bytes are in the passed string and then computed the length of base64 encoding of these bytes with is `ceil(4/3*n)`. With this implementation, the script will send ~ `1/3` more data than configured.